test/test_regression_cli.py

```diff
@@ -46,7 +46,6 @@ def get_csv_md5(dataset, engines, tmpdir):
 
 def setup_module():
     """Update retriever scripts and cd to test directory to find data"""
-    os.chdir("./test/")
```

Comment:
Yes that is true , but for some reason, it seemed like the tests where in that directory`./test/` before. regression tests are run.
When we try to get in to that directory, it will error. I will go through the test and confirm.